presto-product-tests/conf/docker/common/standard.yml

```diff
@@ -0,0 +1,100 @@
+version: '2'
+services:
+
+  java-8-base:
+    image: 'teradatalabs/centos6-java8-oracle:jdk1.8.0_92-80a011f'
+    env_file:
+      - ../../../target/classes/presto.env
+    volumes:
+      - ../../../../:/docker/volumes/presto
+
+  hadoop-base:
+    image: 'teradatalabs/cdh5-hive:2'
+    env_file:
+      - ../../../target/classes/presto.env
+    volumes:
+      - ../../../../:/docker/volumes/presto
+
+  hadoop-kerberos:
```

Comment:
Just noticed that I forgot to delete the Kerberos, MySQL, and Postgres configs from the 'standard' file.  Will push another commit with that change shortly.